src/components/Header/Header.js

```diff
@@ -37,7 +32,7 @@ class Header extends Component {
       <div>
         <Navbar color="light" light expand="md">
         <NavbarBrand className="Logo-icon">
-        <a className="Logo-icon" href="https://communityconnect.netlify.com/"><img src={CClargelogo}/></a>
+        <a className="Logo-icon" href="https://communityconnect.netlify.com/"><img src={CClargelogo} alt="Community Connect logo"/></a>
```

Comment:
The html <a> tag is outputting a major warning in the console. "validateDOMNesting(...): <a> cannot appear as a descendant of <a>."  Instead just put href="/" as a NavbarBrand property to create the link to the main page. Documentation: https://reactstrap.github.io/components/navbar/